modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java

```diff
@@ -166,6 +166,19 @@ public String getConnectionString() {
     protected void configure() {
         super.configure();
 
+        addExposedPorts(
```

Comment:
Interestingly the Couchbase module seems to have always relied on an `EXPOSE` statement + all-ports publication.